test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -470,8 +610,13 @@ private class TestController
         {
             public string UnmarkedProperty { get; set; }
 
-            [NonValueProviderBinderMetadata]
-            public string NonValueBinderMarkedProperty { get; set; }
+            public string[] ArrayProperty { get; } = new string[] { "goodbye" };
+
+            public ICollection<string> CollectionValueBinderMarkedProperty { get; } = new List<string> { "goodbye" };
```

Comment:
don't care much but it's true existing `[ValueProviderMetadata]` isn't relevant in this test class (anymore?)